regulation-worker/internal/delete/api/api_test.go

```diff
@@ -670,6 +776,37 @@ func (cpRespProducer *cpResponseProducer) mockCpRequests() *chi.Mux {
 			return
 		}
 	})
+
+	srvMux.HandleFunc("/workspaces/{workspaceId}/destinations/{destinationId}/authStatus/toggle", func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPut {
+			w.WriteHeader(http.StatusNotFound)
```

Comment:
Wouldn't "Method Not Allowed" make more sense? I know it's a test, but is this test reflecting real behaviour?

```suggestion
			w.WriteHeader(http.StatusMethodNotAllowed)
```